terraso_backend/apps/soilproj/models.py

```diff
@@ -0,0 +1,73 @@
+# Copyright © 2021-2023 Technology Matters
```

Comment:
Issue: The models in this file are missing the `rules_permissions` definition, see: https://github.com/techmatters/terraso-backend/blob/main/terraso_backend/apps/core/models/groups.py#L84

Reply:
I think they have `rules_permissions`, but I noticed reading the docs a bit more that if I use `change`, `delete`, etc. this will be handled in the model validation, is that right?

Reply:
@david-code As far as I understand, this is the model validation, or what do you mean by model validation?